drafts/navigating_concurrency.md

```diff
@@ -0,0 +1,43 @@
+publishAs: tiju
+hideFromHashnodeCommunity: false
+--- 
+
+As described in Demystifying Concurrency: Essential Clarifications, concurrency is evolving. It is clear that programming languages are converging on the async/await Task-based pattern for handling idle time. Even new experimental languages like [Roc](https://www.roc-lang.org/tutorial#the-!-suffix) and [Gleam](https://hexdocs.pm/gleam_otp/gleam/otp/task.html#await) are adopting and improving this pattern.
+
+![async timeline](https://cdn.hashnode.com/res/hashnode/image/upload/v1720084949983/7137b347-2846-458a-8137-d6d04dfbfada.png align="center")
+
+The async/await approach is optimized for the sequential part of asynchronous code execution. Articles like [The Function Colour Myth](https://lukasa.co.uk/2016/07/The_Function_Colour_Myth/) explore the benefits and drawbacks in detail. However, they often overlook the parallel aspect of concurrency.
+
+![](https://cdn.hashnode.com/res/hashnode/image/upload/v1721204475980/829b0131-3f59-44b7-88bc-3229259318b8.png align="center")
+
+One reason for this oversight is that the definitions for [Parallel computing](https://en.wikipedia.org/wiki/Parallel_computing) are generally consistent. There are also well-established patterns for the synchronous part of parallel execution, such as [Data-Parallelism](https://en.wikipedia.org/wiki/Data_parallelism), which have been proven effective, like PLinq in C#. [Single instruction, multiple data (SIMD)](https://en.wikipedia.org/wiki/Single_instruction,_multiple_data) is another method to achieve Data-Parallelism. Also the techniques for executing code on the GPU are well-established and work effectively, as evidenced by Nvidia's success with Cuda in AI applications. It is well understood and reasonably intuitive what this means and why it's challenging.
+
+![](https://cdn.hashnode.com/res/hashnode/image/upload/v1721204331690/62975fa1-a6f1-4dba-ab06-db6c1a5defba.png align="center")
+
+But when it comes to concurrency, there's one more area we haven't discussed yet. This part is often ignored or forgotten, likely because no one has used this visualization to explain it until now 😉. For the parallel async part of concurrency, there doesn't seem to be a clear standard. There are certainly approaches: Go has [Goroutines](https://golangdocs.com/goroutines-in-golang), a variation of [Coroutines](https://en.wikipedia.org/wiki/Coroutine) available in many other programming languages. A similar pattern is explored in Java and other languages with [Green Threads](https://en.wikipedia.org/wiki/Green_thread). Thread and process-based concurrency provided directly by the operating system is also widely used. Additionally, the [Actor Model](https://en.wikipedia.org/wiki/Actor_model) is implemented by many libraries. But non of them seem to be the unified approach that can handle it really well.
+
+> "The world is still short on languages that deal super elegantly and inherently and intuitively with concurrency"
```

Comment:
the > will add quotes at the beginning and end of the quote, so you will have double double quotes here: 
<img width="845" alt="image" src="https://github.com/user-attachments/assets/0335dec8-8ea2-4a64-9117-d88ed2564b30">

Reply:
![image](https://github.com/user-attachments/assets/2bb191a0-a6dc-4713-a934-3801390f4a58)

For me that seems not the case. Where did you view it?